<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="Close_pushButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>580</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DB_pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>580</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>DB.png</normaloff>DB.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>127</width>
     <height>127</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Cover_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>560</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JD_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>582</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>JD.png</normaloff>JD.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>127</width>
     <height>127</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="DD_pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>DD.png</normaloff>DD.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>127</width>
     <height>127</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="infor_textBrowser">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="local_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>379</y>
     <width>431</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
